jupyterhub_traefik_proxy/redis.py

```diff
@@ -130,6 +136,32 @@ def _register_delete_script(self):
         """
         return self.redis.register_script(_delete_lua)
 
+    @default("_delete_script_2")
```

Comment:
`_delete_script_2` isn't defined in this file.

Reply:
oops, did not mean to commit that. Removed.